client/src/app/FeatureFlags.ts

```diff
@@ -1,3 +1,4 @@
 export const FEATURES_ENABLED = {
   migrationWaves: false,
+  jira: false,
```

Comment:
I think Jira can use the migration waves flag as it already does in the routes file for show/hiding the side nav options.